iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractShuntCompensatorModel.java

```diff
@@ -19,9 +17,6 @@
 
     @Override
     public void setShuntCompensator(ShuntCompensatorImpl shuntCompensator) {
```

Comment:
@MioRtia why the public API of ShuntCompensatorModel is empty? I would move everything except this method in the public API (i.e. outside the wrapper)

Reply:
It shouldn't be in the public API, should it? A shunt compensator model should not be created independently from the shunt for me, this method was just because it was created in the adder (so before the shunt itself is created). The method can be deleted if it is not useful anymore.